db/migrate/20171018192044_create_read_ribbons.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReadRibbons < ActiveRecord::Migration[5.1]
+  def change
+    create_table :read_ribbons do |t|
+#      t.belongs_to :user, foreign_key: true
```

Comment:
Why comment this out to do in a second migration? And can the `create_view` migration get folded in to this one as well?